frontend/src/pages/PhishingEmailInfo/index.js

```diff
@@ -0,0 +1,201 @@
+//@flow
+import React, { Fragment, useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import moment from 'moment';
+import { getDebriefData as getDebrief } from '../../actions/debriefActions';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faThumbsUp,
+  faThumbsDown,
+  faArrowDown,
+  faArrowUp,
+} from '@fortawesome/free-solid-svg-icons';
+
+import {
+  Container,
+  EmailItem,
+  EmailItemField,
+  EmailItemContent,
+  Emphasis,
+  Title,
+  ButtonsContainer,
+  AppButton,
+  BehaviorFieldSuccess,
+  BehaviorFieldNegative,
+  SeeDetails,
+} from './ui';
+import WideHeader from '../../components/Header/WideHeader';
+import { negativeMessages, positiveMessages, sanitizeActions } from './utils';
+
+type Props = {
+  phishingEmails: [*],
+  history: Object,
+  getDebrief: (id: string) => void,
+  match: Object,
+  params: Object,
+  participantUuid: string,
+};
+
+function PhishingEmailInfo({
+  phishingEmails,
+  history,
+  getDebrief,
+  match: {
+    params: { participantUuid },
+  },
+}: Props) {
+  const [hasPrevious, setHasPrevious] = useState(false);
+  const [isShowing, setIsShowing] = useState(false);
+  const [nextLink, setNextLink] = useState(false);
+  const [hasNext, setHasNext] = useState(true);
+  const [page, setPage] = useState(0);
+
+  useEffect(() => {
+    getDebrief(participantUuid);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useEffect(() => {
+    page > 0 ? setHasPrevious(true) : setHasPrevious(false);
+  }, [page]);
+
+  useEffect(() => {
+    page < phishingEmails.length - 1 ? setHasNext(true) : setHasNext(false);
+  }, [page, phishingEmails]);
+
+  useEffect(() => {
+    page === phishingEmails.length - 1 ? setNextLink(true) : setNextLink(false);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [page]);
```

Comment:
You don't need a state for this. Just do the logic in the render.

E.g.

```
{page < phishingEmails.length && (
   <AppButton onClick={() => nextPage(page, phishingEmails.length)}>
      Next Email
   </AppButton>
)}
```

Remove the state for previous, next email and next page